modules/members.js

```diff
@@ -0,0 +1,101 @@
+// @ts-check
+
+const Augur = require("augurbot-ts"),
+  u = require("../utils/utils"),
+  c = require("../utils/modCommon"),
+  Discord = require("discord.js"),
+  config = require("../config/config.json");
+
+/**
+ * Creates a profile card - a PNG that contains some user information in a fun format!
+ * @param {Discord.GuildMember} member The member to create the profile card for.
+ * @returns {Promise<Buffer|undefined>} File-like object to attach to your response.
+ */
+async function makeProfileCard(member) {
+  try {
+    const badgeData = require("../utils/badges"),
+      RankInfo = require("../utils/RankInfo"),
+      Jimp = require("jimp");
+
+    const badgePath = "./media/badges/";
```

Comment:
This variable is only used once and doesn't need to be changed by any devs. Might as well just pass it into the function

Reply:
Removed the variable, and added it to the function directly.